ocm.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/python
+import requests
+
+def call_ocm(endpoint, token, parameters=None):
+    requests.packages.urllib3.disable_warnings()
+    r = requests.get("{}/{}".format("https://api.openshift.com/api/", endpoint),
+            params=parameters, 
+            headers = {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer {}'.format(token),
+                'User-Agent': 'pythonCall'
+                }
+            )
+
+    if r.status_code == 204:
+        return []
```

Comment:
Can we just `return` (with the implicit `None`) here?  A mutable list seems potentially surprising :p

Reply:
Sounds like a great idea to me!